src/presentation/components/atoms/Modals.tsx

```
@@ -17,9 +17,7 @@ export function Modals({ open, setOpen, children }: ModalsProps) {
     <Dialog
       open={open}
       handler={handleOpen}
-      className="bg-white rounded-lg shadow-lg p-6"
-    >
-      {children}
+      className="bg-white rounded-lg shadow-lg p-6" children={undefined} placeholder={undefined} onPointerEnterCapture={undefined} onPointerLeaveCapture={undefined}    >
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)